pkg/bitcoin/address_test.go

```diff
@@ -176,14 +69,9 @@ func TestEncodeAddress(t *testing.T) {
 	// Use this in unit-tests to ensure extended key derivation and address
 	// encoding work together as expected.
 	derivePublicKey := func(extendedKey string, derivation []uint32) []byte {
```

Comment:
You may slightly change the func signature to `func(key string, derivation ...int) []byte` or `func(key string, derivation ...uint32) []byte` allowing simpler calls: `derivePublicKey("my-extended-key", 0, 0)` instead of `derivePublicKey("my-extended-key", []uint32{0, 0})`

Reply:
It's a nice sugar, but Goland IDE is unable to show proper field annotation if I use the `...` - only the first index is detected as `derivation`. I prefer to keep it as-is for now, since it doesn't change a lot.